# OdometryList.msg
nav_msgs/Odometry[] odometry_list